src/System.Net.NetworkInformation/src/System/Net/NetworkInformation/IPGlobalProperties.cs

```diff
@@ -61,7 +61,6 @@ internal static IPGlobalProperties InternalGetIPGlobalProperties()
 
         public abstract IPGlobalStatistics GetIPv6GlobalStatistics();
 
-        /// Returns a list of all unicast IP addresses after ensuring they are all stable.
```

Comment:
Maybe we should open a discussion and settle on what we want to do with the comments. I remember @pgavlin replacing all with comments and removing the XML.
I don't care much either way but I do care about being consistent.